repository_service_tuf/cli/admin/ceremony.py

```diff
@@ -95,11 +112,21 @@ def ceremony(save) -> None:
     _add_root_signatures_prompt(root_md, None)
 
     ###########################################################################
-    # Dump payload
-    # TODO: post to API
-    if save:
-        metadatas = Metadatas(root_md.to_dict())
-        settings = Settings(roles)
-        payload = CeremonyPayload(settings, metadatas)
-        json.dump(asdict(payload), save, indent=2)
-        console.print(f"Saved result to '{save.name}'")
+    metadatas = Metadatas(root_md.to_dict())
+    roles_settings = Settings(roles)
+    bootstrap_payload = CeremonyPayload(roles_settings, metadatas)
+    # Dump payload when the user explicitly wants or doesn't send it to the API
+    if out:
+        save_payload(out.name, asdict(bootstrap_payload))
```

Comment:
We use `save_payload` as it handles an error and it as a `click.ClickException` like this:
<img width="696" alt="image" src="https://github.com/repository-service-tuf/repository-service-tuf-cli/assets/16246778/9b7f1360-8776-4fea-b38d-dcbc0ca7db3d">

Instead of:
<img width="997" alt="image" src="https://github.com/repository-service-tuf/repository-service-tuf-cli/assets/16246778/3ab28094-0095-4feb-8768-d6f633352049">

Now, whether we want to add `.json` is another question.
I think it's okay given that we check if `filename` ends with `.json`.

Reply:
`save` is already a file-like object. It's a hack to take its name and open a different file object. It also bypasses all the neat [click file magic](https://click.palletsprojects.com/en/8.1.x/arguments/#file-arguments). This includes raising `click.FileError`, which would give you the desired formatting automatically.

If you want to open the file yourself, then you should make the argument a `click.Path` type instead. But I really don't see the advantage.

If you want to keep the `.json` suffix, you should change the docs.

Reply:
I thought about it and given that most of the errors we could have are with file openings, then probably it makes sense to relly on click for that.
If we find other errors not handled from click we can have a helper such as `save_payload` in the future.
For now, I will not remove `save_payload` as it's used from admin-legacy.